### ISSUE TITLE
[question]can i use a keras model as the feature_extractor_class of algorithm PPO?

### ISSUE THREAD

**Issue by wujiren** [DLR-RM/stable-baselines3#104]
in the stable-baselines tf version, if i use a Keras model, the PPO algorithm would not work, maybe due to the variable_scope. still i need to use some module in github written in Keras, in this version, can i use these Keras model as the feature_extractor_class of PPO? and what about other algorithm such as A2C?

**Comment by Miffyli** [comment]
I do not think you can backpropagate gradients from Keras models to PyTorch models. If you do not need gradient updates (i.e. only inference), then you could embed the Keras model inside the environment which then returns those features as observations. 

Another approach could be to re-create the model in PyTorch manually and import the weights from the Keras model, but you have to be careful that the operations match between Keras and PyTorch.

**Comment by wujiren** [comment]
get it. thanks
